frontend/src/components/LandingPage.js

```diff
@@ -4,57 +4,64 @@ import { Link } from "react-router-dom";
 import Header from "./Header";
 import BalancePresentation from "./BalancePresentation";
 import Footer from "./Footer";
+import "../styles/LandingPage.css";
 
 function Landing() {
   return (
-    <>
+    <div className="LandingPage">
       <Header />
       <main>
         <BalancePresentation />
         <Accordion>
           <Card>
             <Accordion.Toggle as={Card.Header} eventKey="0">
-              Update salary
+              <Button className="w-100 text-white">Update Salary</Button>
             </Accordion.Toggle>
             <Accordion.Collapse eventKey="0">
               <Card.Body>
                 <Form>
-                  <Form.Group controlId="formSalary">
-                    <Form.Label>Salary</Form.Label>
-                    <Form.Control type="number" /> €
-                  </Form.Group>
-                  <Button variant="primary">Save</Button>
+                  <div className="form-wrapper">
+                    <Form.Group controlId="formSalary">
+                      <Form.Label>Salary</Form.Label>
+                      <Form.Control type="number" />
+                      <p>€</p>
```

Comment:
The \<p\> wrapper is probably not necessary (again, polluting dom unnecessarily).

Reply:
If i dont give p tag, how i can style "€" this sign?

Reply:
We can refactor this also later. Ideally placement should be done on the container level rather than per each element.

Reply:
"Style refactoring" PBI added, resolving this discussion.